src/java/main/org/apache/zookeeper/server/SessionTracker.java

```diff
@@ -47,21 +47,20 @@
     long createSession(int sessionTimeout);
 
     /**
-     * Add a global session to those being tracked.
+     * Track the session expire, not add to ZkDb.
      * @param id sessionId
      * @param to sessionTimeout
      * @return whether the session was newly added (if false, already existed)
      */
-    boolean addGlobalSession(long id, int to);
+    boolean trackSession(long id, int to);
 
     /**
-     * Add a session to those being tracked. The session is added as a local
-     * session if they are enabled, otherwise as global.
+     * Add the session to the under layer storage.
```

Comment:
I think the comment here is a little bit confusing. `commitSession`itself does not directly add session to storage, it's the snapshot that does this. I think the comment should mention that this function will add the session to the zkDB, this also maps to the `trackSession` comment (which does not modify zkDB.). It would be great if a reader can instantly get what `commit` means here by just reading the comment.

Reply:
In LocalSessionTracker, commitSession is used to update the in memory local session map, which is not in zkDB. How about change it to:

"Add the session to the local session map or global one in zkDB."

Reply:
Sounds good to me with updated comment.